.larasail/database

```diff
@@ -36,6 +38,22 @@ create_database_details() {
     echo "[client]" >> "/etc/.larasail/tmp/mysql.cnf"
     echo "password=$ROOT_PASSWORD" >> "/etc/.larasail/tmp/mysql.cnf"
 
+    if $DATABASE_USER_PROMPT; then
+        printf "Please enter ${Yellow}database user${Color_Off} to create [${Cyan}$DATABASE_USER${Color_Off}]: "
+        read ANSWER
+        if ! [ -z "$ANSWER" ]; then
+            DATABASE_USER="$ANSWER"
+        fi
+    fi
+
+    if $DATABASE_NAME_PROMPT; then
+        printf "Please enter ${Yellow}database name${Color_Off} to create [${Cyan}$DATABASE_NAME${Color_Off}]: "
+        read ANSWER
+        if ! [ -z "$ANSWER" ]; then
+            DATABASE_NAME="$ANSWER"
+        fi
+    fi
+
```

Comment:
Added interactive prompts to ask the user when flags are not specified explicitly.